front/src/applications/stdcmV2/components/StdcmInputVia.tsx

```diff
@@ -9,39 +9,44 @@ import { ISO8601Duration2sec } from 'utils/timeManipulation';
 
 import { StdcmStopTypes } from '../types';
 
+type StdcmInputViaProps = {
+  stopType?: StdcmStopTypes;
+  pathStepStopFor: PathStep['stopFor'];
+  updatePathStepStopTime: (stopTime: string) => void;
+};
+
 const StdcmInputVia = ({
   stopType,
-  pathStep,
+  pathStepStopFor,
   updatePathStepStopTime,
-}: {
-  stopType?: StdcmStopTypes;
-  pathStep: PathStep;
-  updatePathStepStopTime: (stopTime: string) => void;
-}) => {
+}: StdcmInputViaProps) => {
   const { t } = useTranslation('stdcm');
 
-  const [pathStepStopTime, setPathStepStopTime] = useState(
-    pathStep.stopFor ? `${ISO8601Duration2sec(pathStep.stopFor) / 60}` : ''
+  const computedStopTime = useMemo(
+    () => (pathStepStopFor ? `${ISO8601Duration2sec(pathStepStopFor) / 60}` : ''),
```

Comment:
Could you add an util for that ?

```suggestion
    () => (pathStepStopFor ? `${secToMin(ISO8601Duration2sec(pathStepStopFor))}` : ''),
```